src/modifier.ts

```diff
@@ -167,3 +173,178 @@ function pairwiseMerge(modifiers1: Modifiers, modifiers2: Modifiers) {
 export function mergeModifiers(...modifierss: Modifiers[]): Modifiers {
   return modifierss.reduce((a, b) => pairwiseMerge(a, b), {});
 }
+
+function replaceAll(
+  str: string,
+  searchValue: string,
+  replaceValue: string
+): string {
+  const newStr = str.replace(searchValue, replaceValue);
+  if (newStr === str) return newStr;
+  return replaceAll(newStr, searchValue, replaceValue);
+}
```

Comment:
I would either polyfill this, or just use a regexp with the `global` flag. 

By which I mean almost ceratinly the second one.

Reply:
Yeah this is nutso sadly